frontend/src/types/Common.ts

```diff
@@ -8,6 +8,9 @@ export type RawDate = string;
 
 export type KioskMode = string;
 
+// cluster name to denote the cluster where Kiali is deployed
+export const HomeClusterName = '_kiali_home';
+
```

Comment:
This is kind of confusing to display to users since their actual cluster name probably won't be `_kiali_home`. This `_kiali_home` var wasn't really meant to be an external facing detail. We just needed something to put in the clients map that wouldn't overlap with another valid cluster name. On the backend we should probably get the "real" cluster name from something like the istio configmap but regardless of how the backend populates this value the cluster names should come from the backend. Why is it necessary to have this in the frontend as well?

Reply:
if we have this value in backend (not hardcoded), then that can be reused in frontend, but in backend it is hardcoded as well.

Reply:
Can the frontend optionally send cluster as a query param to the backend and when there's no cluster query param, the backend can assume that you want the home cluster? This way single cluster stays the same and we don't need to copy this hard coded value to the frontend that will probably be going away soon. If you are on a list page then the backend should populate the cluster names already and if you are on a details page then (I think) the url will contain the cluster name?

Reply:
the frontend now has empty default cluster name in params, and the backend in this case takes from the default value it has